src/cli/commands/cache.js

```diff
@@ -64,3 +64,7 @@ export const {run, setFlags} = buildSubCommands('cache', {
     }
   },
 });
+
```

Comment:
It's kind of mixed now: https://github.com/yarnpkg/yarn/blob/master/src/cli/commands/config.js#L77
But I'll move both.